packages/helpers/classes/attachment.js

```diff
@@ -39,17 +40,29 @@ class Attachment {
     data = toCamelCase(data);
 
     //Extract properties from data
-    const {content, filename, type, disposition, contentId} = data;
+    const {content, filename, type, disposition, contentId, file} = data;
+
+    let fileContent = !file ? content : this.readFile(file);
```

Comment:
File content is immutable so can be `const`. I'm not sure about the logic either though. If content is passed, should that take priority over a file? Maybe a warning if both are passed?

Reply:
I think that the file should take priority over content, and warn the user if both is passed is a good idea.

Reply:
Is there any example of how to give the user a warning here?

Reply:
Actually we don't show warnings anywhere really, just throw errors. Maybe an error is suitable, because you are not expected to pass both values. Then, we also don't need to worry about what takes priority.